src/content/tutorials/drizzle-with-db/drizzle-with-neon.mdx

```diff
@@ -0,0 +1,241 @@
+import { pgTable, integer, serial, text, timestamp } from "drizzle-orm/pg-core";
+
+export const authors = pgTable("authors", {
+  id: serial("id").primaryKey(),
+  name: text("name").notNull(),
+  bio: text("bio"),
+  createdAt: timestamp("created_at").notNull().defaultNow(),
+});
+
+export const books = pgTable("books", {
+  id: serial("id").primaryKey(),
+  title: text("title").notNull(),
+  authorId: integer("author_id").references(() => authors.id),
+  createdAt: timestamp("created_at").notNull().defaultNow(),
+});
+```
+
+The code defines two tables: `authors`, which will contain the list of all the authors, and `books`, which will contain the list of books written by the authors. Each book is associated with an author using the `authorId` field.
+
+#### Generate migrations
+
+We'll use the drizzle-kit CLI tool. Run the following command in your terminal. 
+
+```bash
+npx drizzle-kit generate:pg --schema src/schema.ts --out ./drizzle
+```
+
+These migrations are stored in the `drizzle`  directory, as specified above. This directory will contain the SQL files necessary to update your database schema and a `meta` folder for storing snapshots of the schema at different migration stages. 
+
+Example of the generated migration:
+
+```sql
+CREATE TABLE IF NOT EXISTS "authors" (
+	"id" serial PRIMARY KEY NOT NULL,
+	"name" text NOT NULL,
+	"bio" text,
+	"created_at" timestamp DEFAULT now() NOT NULL
+);
+--> statement-breakpoint
+CREATE TABLE IF NOT EXISTS "books" (
+	"id" serial PRIMARY KEY NOT NULL,
+	"title" text NOT NULL,
+	"author_id" integer,
+	"created_at" timestamp DEFAULT now() NOT NULL
+);
+--> statement-breakpoint
+DO $$ BEGIN
+ ALTER TABLE "books" ADD CONSTRAINT "books_author_id_authors_id_fk" FOREIGN KEY ("author_id") REFERENCES "authors"("id") ON DELETE no action ON UPDATE no action;
+EXCEPTION
+ WHEN duplicate_object THEN null;
+END $$;
+```
+
+#### Applying changes to the database
+
+Create a `migrate.ts` file and add the code below. 
+
+```typescript copy filename="src/migrate.ts"
+import { db } from './db';
+import { migrate } from "drizzle-orm/neon-http/migrator";
+import { config } from "dotenv";
+
+config({ path: ".env" });
+
+const main = async () => {
+  try {
+    await migrate(db, { migrationsFolder: "drizzle" });
+    console.log("Migration completed");
+  } catch (error) {
+    console.error("Error during migration:", error);
+    process.exit(1);
+  }
+};
+
+main();
+```
+
+To execute the migration scripts, run the following command. 
+
+```bash
+tsx ./src/migrate.ts
+```
+
+Learn more about the [migration process](https://orm.drizzle.team/docs/migrations).
+
+</Steps>
+
+
+### Basic file structure
+
+This is the basic file structure of the project. In the `src/db` directory, we have database-related files including connection in `db.ts`, schema definitions in `schema.ts`, and a migration script in `migrate.ts` file which is responsible for applying migrations that stored in the `migrations` directory.
```

Comment:
You can remove part about `migrate.ts` cause we use `drizzle-kit migrate` command for applying migrations